customized invitation

seanmarcia opened the pull request:
This PR resolves #1028 

This adds in an `invitation_text` field on the organization that can be edited/updated on the organization manage page.

It then sends the text, if present along with the other invitation to partnerbase when creating a new partner/user there. If the text is empty it uses the default text.

There is a corresponding PR on the partner side.

seanmarcia commented:
This has been merged on the partner side btw.

mdworken commented on app/jobs/update_diaper_partner_job.rb:
```diff
@@ -5,7 +5,8 @@ class UpdateDiaperPartnerJob
   def perform(partner_id)
     @partner = Partner.find(partner_id)
-    @response = DiaperPartnerClient.post(@partner.attributes) if Flipper.enabled?(:email_active)
+    @invitation_message = @partner.organization.invitation_text.to_plain_text
```
Can `to_plain_text` be safely called on nil?

mdworken replied:
Can to_plain_text be safely called if `invitation_text` is nil?

mdworken commented on yarn.lock:
```diff
@@ -76,31 +65,27 @@ acorn@^5.0.0:
     json-schema-traverse "^0.3.0"
 
 ajv@^6.1.0, ajv@^6.5.5:
-  version "6.10.2"
```
Looks like a lot of these version changes are downgrades - was committing this file intentional?

seanmarcia replied:
My bad on this. I think when I reset things to pull down the prod databases to debug Rachel's issue this happened!

seanmarcia commented:
I made a few changes to the PR. I don't think it makes sense to use the rich text editor if we're only passing text across to the partner app so I changed the field to just a text area. I also updated the labels the fields to make them more intuitive for the diaper banks. No instead of "Invitation Message" and "Default Text" I changed them to "Custom Partner Invitation Message" and "Default Text Included In Distribution Emails".

![](https://i.imgur.com/hXEVrbE.png)
